tests/api_tests/user_tests.py

```diff
@@ -14,10 +14,10 @@ def setUp(self):
         self.app = app.test_client()
         db.create_all()
         self.user_role = RoleFactory()
-        user = UserFactory.create(role=self.user_role)
+        user = UserFactory.create(roles=[self.user_role])
```

Comment:
For consistency, in subscription tests, you used: self.user = UserFactory(roles=[role],subscribed=[feed]).

Let's either remove the .create or add it to the others.